### PR TITLE
[Release-2021-03] chore: update webhook docs

### PR DESCRIPTION
**Pull request by mokimo**
Add new webhooks according to https://github.com/livingdocsIO/livingdocs-server/pull/3359

### CODE REVIEW

File: reference-docs/server-configuration/webhooks.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # Webhooks
-- added in [`release-2020-05`](https://github.com/livingdocsIO/livingdocs-release-notes/blob/master/releases/release-2020-05.md)
+- added `document.published` and `document.publish` added in [`release-2020-05`](https://github.com/livingdocsIO/livingdocs-release-notes/blob/master/releases/release-2020-05.md)
```
Comment (benib) [inline reference-docs/server-configuration/webhooks.md:2]:
```suggestion
- The feature and the two events `document.published` and `document.publish` were added in [`release-2020-05`](https://github.com/livingdocsIO/livingdocs-release-notes/blob/master/releases/release-2020-05.md)
```

File: reference-docs/server-configuration/webhooks.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
+- all other events added in `release-2021-03`
```
Comment (benib) [inline reference-docs/server-configuration/webhooks.md:3]: Do we have a list somewhere with all these events? Would make sense to link it here, or list them directly.

**Review by benib — CHANGES_REQUESTED** [review]

**Comment by peyerluk** [comment]
Release-2020-02? :)

File: reference-docs/server-configuration/webhooks.md (hunk 2 above)
Comment (mokimo) [reply reference-docs/server-configuration/webhooks.md]: Thanks for the two suggestions. Committed your suggested change & added a list with the webhooks that were added in the `release-2021-03`

**Pull request merged**